ocs_ci/framework/exceptions.py

```diff
@@ -3,3 +3,17 @@ class ClusterPathNotProvidedError(Exception):
 
     def __str__(self):
         return "Please provide a --cluster-path that is empty or non-existant."
+
+
+class ClusterNameLengthError(Exception):
+
+    def __init__(self, name, min, max):
+        self.name = name
+        self.min = min
```

Comment:
Would it be useful to default to using `CLUSTER_NAME_MIN_CHARACTERS` and `CLUSTER_NAME_MAX_CHARACTERS` if `min`/`max` are not passed in to the `__init__`? This way we don't need pass the values through to the error init if we are using our defaults anyways.

Reply:
Yes, make sense. I'll adapt it accordingly.